F/FluxPruning/Deps.toml

```diff
@@ -0,0 +1,14 @@
+[0]
+CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
+ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+DocumenterTools = "35a29f4d-8980-5a13-9543-d66fff28ecb8"
```

Comment:
Doesn't look Documenter is used anywhere in the code of your package: https://github.com/search?q=repo%3AEBothereau%2FFluxPruning.jl%20documenter&type=code.  That shouldn't be a dependency.